ABRSchema_0-2-0.json

```diff
@@ -634,6 +676,69 @@
                     }
                 },
                 "additionalProperties": false
+            },
+            "Volumes": {
```

Comment:
Meta-comment for this section - we should add descriptions for each of these inputs - similar to [this line](https://github.com/ivlab/abr-schema/blob/master/ABRSchema_0-2-0.json#L527).